src/pyopticon/dashboard.py

```diff
@@ -152,6 +157,39 @@ def add_interlock(self, fn):
         :type fn: function
         """
         self.all_interlocks.append(fn)
+
+    def notify(self, event, modifier =None):
+        """ Updates observers when there is an event logged. If an observer created the event, they are not updated.
```

Comment:
I suggest changing this to `notifier` so it's a bit more clear, but don't feel super strongly on it.